operator/gitops/argocd/pipeline-service/metrics-exporter/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ resources:
 images:
   - name: quay.io/redhat-appstudio/pipeline-service-exporter
     newName: quay.io/redhat-appstudio/pipeline-service-exporter
-    newTag: 42a1938a18bf5b999241fafbf0d57a1c7d19e0f2
+    newTag: cdff9b19bf4b6e7009412ae6103be882a3a36f9b
```

Comment:
the cdff9b19bf4b6e7009412ae6103be882a3a36f9b tag is up at https://quay.io/repository/redhat-appstudio/pipeline-service-exporter?tab=tags